recipe/meta.yaml

```diff
@@ -21,20 +21,18 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

Reply:
I second this suggestion

Reply:
can you elaborate on why this switch is needed here? I understand more or less what it does, but I'm wondering why '--no-deps --no-build-isolation' is not in every yaml.